core/trino-spi/pom.xml

```diff
@@ -40,6 +40,12 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
It is intentional for the SPI module to not depend on Guava. We don’t want to force such a dependency on plugins, which may need to import incompatible versions of the library.

Reply:
Thanks for your review, I will close this PR.